apps/workflows/api.py

```diff
@@ -128,6 +126,88 @@ def post(self, request, flaw_id):
             return Response({"errors": str(e)}, status=status.HTTP_409_CONFLICT)
 
 
+class RevertWorkflow(RudimentaryUserPathLoggingMixin, APIView):
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="Jira-Api-Key",
+                type=str,
+                location=OpenApiParameter.HEADER,
+                description="User generated api key for Jira authentication.",
+            ),
+            OpenApiParameter(
+                name="Bugzilla-Api-Key",
+                type=str,
+                location=OpenApiParameter.HEADER,
+                description="User generated api key for Bugzilla authentication.",
+            ),
+        ]
+    )
```

Comment:
I see this kind of OpenApi parameters already used several places in this file, can we do something similar as in [`osidb/api_views.py`](https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/api_views.py#L382)?